### DISCUSSION TITLE
How to ignore the image synchronisation ?

### DISCUSSION THREAD

**Discussion by ShivamJoker** [wix/Detox#3028] [Q&A] [answered]
I tried using the URLBlocker as an app launch argument with `domain.com/*` but it didn't go forward still kept waiting for the images to load.

**Comment by d4vidi** [comment, marked as answer]
Were you referring to this API? https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#deviceseturlblacklisturls

**Reply by ShivamJoker** [reply]
@d4vidi yes. That only I am not able to ignore image synchronisation

**Reply by d4vidi** [reply]
Try using a complete (`http://...`) and specific URL with no wildcards, and see whether that helps.

**Reply by ShivamJoker** [reply]
I think I did it with https only and without the / at the end. I'll try again.

I'm using iOS BTW

**Reply by ShivamJoker** [reply]
> Try using a complete (`http://...`) and specific URL with no wildcards, and see whether that helps.

It seems to be working but in example the wildcards were given, shouldn't we update the docs ?
